### ISSUE TITLE
[BUG] Can't connect through wired ethernet

### ISSUE THREAD

**Issue by muchachagrande** [nextcloud/talk-android#3896]
### Steps to reproduce

1. Open Talk app
2. Observe the popup asking to check Internet connection

### Expected behaviour

The app should open to show the chat list.

### Actual behaviour

A popup appears with the message: "(!) Please check your Internet connection" allowing to press "Update" or "Close" buttons.
Pressing "Update" just do nothing and pressing "Close" closes the popup and it leaves you on the empty chat list.

### Device brand and model

Raspberry Pi 4

### Android version

34 - Lineage 21 (Android 14) [without GPlay]

### Nextcloud Talk app version

19.0.0

### Nextcloud server version

28.0.5

### Talk version

18.0.7

### Custom Signaling server configured

Yes (specify version in Additional Information)

### Custom TURN server configured

Yes

### Custom STUN server configured

Yes

### Android logs

No log

### Server log

```bash
No log
```

### Additional information

Signaling version is 1.2.4

The app is not recognizing the wired ethernet connection. This happened also with Lineage 19 yesterday and it pushes me to update the panel to Lineage 21 but it produced the same error.
Previous version of Talk, 18.1.0 worked fine on Lineage 19.

This is the screenshot of the opened app with the popup dialogue:

![Captura desde 2024-05-06 12-20-16](https://github.com/nextcloud/talk-android/assets/20299175/ab130cc5-e2e7-400b-b6b6-842b39ad47ea)

**Comment by sppg** [comment]
Hello,
  We are having this same issue on a Samsung phone with talk 19.0.0 and 19.0.1, it was working fine with the previous versions.
Is there any workaround waiting the patch to correct it please?
Thank you

**Comment by muchachagrande** [comment]
Not yet... I tried to connect through a proxy but it shows the same message "Please check your internet connection".

**Comment by sppg** [comment]
I noticed that if I receive a message and click on the notification, the app opens correctly the chat and I can send and receive.

**Comment by muchachagrande** [comment]
I can't receive notifications. I'm using f-droid version. Also I don't have 19.0.1 yet.

**Comment by mahibi** [comment]
> I noticed that if I receive a message and click on the notification, the app opens correctly the chat and I can send and receive.

makes sense as the check only takes place in ConversationListActivity for now. Will be improved that it's done on overall level
